R/finalize_plot.R

```diff
@@ -554,8 +572,10 @@ buildChart <- function(plot,
   if (number_of_legends > 1) {
     # if multilegend plot, establish loop to change margins
     for (i in 1:(number_of_legends-1)) {
```

Comment:
How does this react to a plot with no legend? I think there is a "step through" function that is preferable here